src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/DisableAzureRmAlias.cs

```diff
@@ -0,0 +1,188 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.Commands.Profile.AzureRmAlias
+{
+    /// <summary>
+    /// Cmdlet to clear default options. 
+    /// </summary>
+    [Cmdlet("Disable", "AzureRmAlias", SupportsShouldProcess = true)]
+    [OutputType(typeof(string))]
+    public class DisableAzureRmAlias : AzureRMCmdlet
+    {
+        [Parameter(Mandatory = false, HelpMessage = "Indicates what scope aliases should be disabled for.  Default is 'Process'")]
+        [ValidateSet("Process", "CurrentUser", "LocalMachine")]
+        public string Scope { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Indicates which modules to disable aliases for. If none are specified, default is all enabled modules.")]
+        public string[] Module { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "If specified, cmdlet will return all disabled aliases")]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            string jsonmapping = Mappings.jsonMappings;
```

Comment:
I think if we factor this out into common mappings code, and factor the profile manipulation into a helper class, it will be easier to mock and unit test this cmdlet